### PR TITLE
 Add new PNG images to Comp Sci

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Add new PNG images that work on both the dark and light mode in the Enki App

Need review for:
1. Some images have all circles highlighted(should be either all or none), which is better?
2. The `comp-sci/data-structures-and-algorithms/intro-data-structures/the-queue-data-structure.md` (alt-description) image doesn't want to convert to an SVG on the enkisvg converter, but it works on codepen

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/comp-sci/data-structures-and-algorithms/algorithms-i/approximating-memory-and-time-required-by-data-types.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/comp-sci/data-structures-and-algorithms/algorithms-i/approximating-memory-and-time-required-by-data-types.md?ref=b0497652c1566cacc8d35e8431b5936930ae3f7d)

**Comment by kapnobatai137** [comment]
I'll test these images in Spider and see what is wrong with the image in The Queue Data Structure this Tuesday.

**Comment by kapnobatai137** [comment]
- checked in spider, most of the images look good
- some images have had their background changed from white to orange, when they background color (`fill="#FFF"`) should've been removed completely

File: comp-sci/data-structures-and-algorithms/intro-data-structures/the-array-data-structure.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ parent: what-is-a-data-structure
 
 An **array** is a data structure organized as a linear collection of elements, for which an **index** (a numerical indicator of position) is used to access its content. The same information may appear in the array more than once but under a different index. This is how you could picture it:
 
-![array](https://img.enkipro.com/630c4678f67993a0479b25467cda6db9.png)
+![array](https://img.enkipro.com/0e755de4b3d32ffb01a139a3904e8d2d.png)
```
Comment (kapnobatai137) [inline comp-sci/data-structures-and-algorithms/intro-data-structures/the-array-data-structure.md]: here, the background of the squares shouldn't be orange

File: comp-sci/data-structures-and-algorithms/intro-data-structures/the-linked-list-data-structure.md [hunk 2]
```diff
@@ -41,11 +41,11 @@ However, you would still need to count to get to position `i`: no one has a sign
 
 The *conga line* is an example of a **SinglyLinkedList** because you cannot see who is behind you. The last one (in the example below, *G*) still has a reference that points to nothing(or `null`):
 
-![sll](https://img.enkipro.com/7b0cad245c9ed5cb55d6c12678629acd.png)
+![sll](https://img.enkipro.com/41149a17a426e6eea8d173a2284b3f47.png)
```
Comment (kapnobatai137) [inline comp-sci/data-structures-and-algorithms/intro-data-structures/the-linked-list-data-structure.md]: background inside squares shouldn't be orange

File: comp-sci/data-structures-and-algorithms/intro-data-structures/the-linked-list-data-structure.md [hunk 3]
```diff
@@ -41,11 +41,11 @@ However, you would still need to count to get to position `i`: no one has a sign
...
 
 To make it a **DoublyLinkedList**, everyone would have to use one hand to hold the person in front of them, and their other hand to hold the person behind them. In this case, you would be able to move through in both directions, from *start to finish* and from *finish to start*.
 
-![dll](https://img.enkipro.com/46b5d724b14755373af739d2b55fc878.png)
+![dll](https://img.enkipro.com/1f254e3259fe47923f73f1dbc78d9bd6.png)
```
Comment (kapnobatai137) [inline comp-sci/data-structures-and-algorithms/intro-data-structures/the-linked-list-data-structure.md]: background inside squares shouldn't be orange

File: comp-sci/data-structures-and-algorithms/intro-data-structures/the-stack-data-structure.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ A **stack** is a collection of objects optimized for two things:
 
 The items are inserted and removed according to the *last-in, first-out* (*LIFO*) principle.
 
-![stack](https://img.enkipro.com/9a05488e0127141d09e7bf0d600f8407.png)
+![stack](https://img.enkipro.com/df45877c1de4e1891f45d451089edc3e.png)
```
Comment (kapnobatai137) [inline comp-sci/data-structures-and-algorithms/intro-data-structures/the-stack-data-structure.md]: background inside squares shouldn't be orange.

here is how the image looked before https://img.enkipro.com/9a05488e0127141d09e7bf0d600f8407.png

**Comment by kapnobatai137** [comment]
the above are the ones I've noticed when doing the first workout in spider, but I suspect there are a lot more. to be clear, I am not talking about nodes/squares that were **supposed to be** evidentiated  using the orange color. I am talking about nodes/squares that used to have a white background and that was changed to orange, like in the examples above (you can look at the history of each file if you are unsure).

what should be done is looking at the svg code and:
1. finding the elements that used to have a white background (`fill="#FFF"`) and now have an orange background
2. delete the `fill="orange"` attribute, and now the background will be transparent in both white and dark mode

@Stefan-Stojanovic

File: comp-sci/data-structures-and-algorithms/intro-data-structures/the-array-data-structure.md (hunk 1 above)
Comment (Stefan-Stojanovic) [reply comp-sci/data-structures-and-algorithms/intro-data-structures/the-array-data-structure.md]: They were colored when they were made but on the 2nd version(the one before this pr) they aren't. So I wasn't sure which version to do. The colors of the original versions are exactly as in this PR,

Comment (Stefan-Stojanovic) [reply comp-sci/data-structures-and-algorithms/intro-data-structures/the-array-data-structure.md]: Should I do them without the coloring then?

**Comment by kapnobatai137** [comment]
update:

some images proved to be more problematic and my general approach would have not worked. I have personally verified all the images and updated the ones that required changing.

**Comment by kapnobatai137** [comment]
image
image
image
image

**Pull request merged**